packages/@react-aria/virtualizer/src/useTabbableChild.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {getFocusableTreeWalker} from '@react-aria/focus';
+import {RefObject, useEffect, useState} from 'react';
+
+export interface TabbleChildAria {
+  /** Indicator of if the element can be focused. */
+  tabIndex?: number | undefined
+}
+
+// This is based/coped from useTabPanel.ts
+export function useTabbableChild(ref: RefObject<HTMLElement>): TabbleChildAria {
```

Comment:
maybe instead of this, we do something like
```suggestion
export function useHasTabbableChild(ref: RefObject<HTMLElement>): boolean {
```
I think it'll be easier to read in places where it's used and then components/hooks using it can make their own decision about what to do if there is a focusable/tabbable child.

Reply:
This factor looks better, thanks for suggesting it.